_extensions/posit-docs/theme-dark.scss

```diff
@@ -0,0 +1,395 @@
+$posit-dark-burgundy-1:#78384F;
+$posit-dark-burgundy-2:#542938;
+$posit-dark-burgundy-3:#2E171F;
+$primary: $posit-teal;
+$dark-md-body: $posit-light-gray-1;
+$dark-md-background: #0C1721;
+
+// Feature preview heading colors
+$preview-header: $posit-orange; /* posit orange, contrast: 8.45 */
+$preview-header-border: darken($preview-header, 5%);
+
+// scss-docs-end color-variables
+
+// Typography
+// Font, line-height, and color for body text, headings, and more.
+
+//scss-font-start font-variables
+$font-family-sans-serif: "Open Sans", "Segoe UI", Roboto, "Helvetica Neue", "Noto Sans", "Liberation Sans", Arial, sans-serif, system-ui;
+$font-family-monospace: "Source Code Pro", monospace;
+$font-family-emoji: "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji";
+$font-size-base: 1rem;
+// scss-docs-end font-variables
+
+//font and body color
+
+$body-color: $dark-md-body;
+$popover-bg: $posit-green !default; // check this - citation previews
+
+// scss-docs-start font-sizes - seems that I have to keep these in the style sheet?
+$h1-font-size:44px;
+$h2-font-size:28px;x
+
+h1 {
+  size: $h1-font-size;
+}
+h2 {
+  size: $h2-font-size;
+}
+// scss-docs-end font-sizes
+
+// Headings
+// scss-docs-start headings-variables
+$headings-color: #ffffff;
+// scss-docs-end headings-variables
+
+//  Customize the navbar
+$navbar-bg: lighten($dark-md-background, 5%);
+$navbar-fg: #ffffff; 
+
+// CSS overrides
+
+// Body
+//
+// Settings for the `<body>` element.
+
+//$body-color: $posit-green !default;
+$body-bg: $dark-md-background !default; // dark mode change
+
+
+// Nav and footer
+
+/* Nav bar */
+
+.navbar-title {
+  color: #ffffff; // dark theme change
+}
+
+.nav-link {
+font-family: $font-family-monospace;
+text-transform: uppercase;
+letter-spacing: .03em;
+font-size: 14px !important;
+font-weight: 500 !important;
+color: #fff !important;
+}
+
+.navbar-nav .nav-link.active,
+.navbar-nav .nav-link.show {
+color: $posit-teal !important;
+font-weight: 700 !important;
+}
+
+/* Nav bar  right hamburger menu */
+
+.dropdown-item:hover, .dropdown-item:focus {
+color: var(--bs-dropdown-link-hover-color);
+background-color: #0F1C29 !important;
+}
+
+/* Footer */
+
+.nav-footer {
+  color:  #ffffff !important;
+  border-top: solid $posit-gray 1px;
+}
+
+/* Footer icons */
+.nav-footer a {
+color: #ffffff !important;
+}
+
+
+// Footnotes
+
+.footnote-back {
+  color: $posit-light-teal-2;
+}
+
+
+// Breadcrumbs 
+
+.quarto-title-breadcrumbs .breadcrumb li:last-of-type a {
+  color: $posit-light-gray-1 !important;
+}
+
+// Page navigation
+
+.nav-page .nav-page-text {
+  color: $posit-light-gray-1;
+}
+
+// Code
+
+/* Code block elements */
+
+pre > code.sourceCode > span {
+  //color: #003B4F;
+}
+
+code span.ex {
+    //font-weight: bold;
+    color: #ffffff;
+}
+
+code span.dt {
+  color: #FFABCA;
+    //font-style: inherit;
+}
+
+code span.ot {
+    //color: rgb(60, 222, 222);
+}
+
+code span.st {
+    //color: greenyellow;
+}
```

Comment:
for the commented-out bits here: do these need to stay (i.e. they are part of something coming soon), or can they be removed?

Reply:
I think that I had added the styles assuming that they would need updates but I think that I am safe to remove